### PR TITLE
Install the sass-loader

### PR DESCRIPTION
**Pull request by rahmot**
To setup the variable for BoostrapVue will need to install the `sass-loader`. So I have installed it and we commit the remaining setups.

### CODE REVIEW

File: src/assets/styles.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import './variable';
```
Comment (bebatut) [inline src/assets/styles.scss:1]: You could maybe copy there the content of the same file on the hub: https://github.com/galaxyproject/galaxy-hub/blob/master/src/assets/styles.scss
Then we could be sure to have a similar interface, similar colors

Comment (bebatut) [reply src/assets/styles.scss:1]: Please then add as a comment in the file that was copied from the Hub

**Review by bebatut — COMMENTED** [review]
Could you also copy the content of `style.scss`, `custom.scss` and `variables.scss` from the hub? 
https://github.com/galaxyproject/galaxy-hub/tree/master/src/assets
And add as a comment in each of the file from where it was copied

Thanks a lot

File: src/assets/styles.scss (hunk 1 above)
Comment (bebatut) [reply src/assets/styles.scss:1]: That should also fix the deployment error

**Comment by rahmot** [comment]
@bebatut, thanks! I have copied the content of `style.scss` and `variables.scss` from the hub. And I rename `customs.scss` to `overides.scss` because I couldn't find the custom.scss file. I also added the `palette.styl`, guess we will need that as well. Kindly let me if there is anything I didn't do correctly. Thanks!

File: src/assets/styles.scss [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+$jumbotron-bg: white;
+$carousel-indicator-border-color: gray;
+$carousel-indicator-active-bg: $navbar-inverse-bg;
+$carousel-control-width: 5%;
+
+@import "~bootstrap/scss/bootstrap.scss";
+
+//@import "~@fortawesome/fontawesome-free/scss/_variables.scss";
+$fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
+@import "~@fortawesome/fontawesome-free/scss/fontawesome.scss";
```
Comment (bebatut) [inline src/assets/styles.scss:44]: This import is failing in the deployment via netlify: https://app.netlify.com/sites/hungry-aryabhata-193bcf/deploys/61fcf438507d8d00080a5ace
You should add `fortawesome` in the `package.json`: see https://github.com/galaxyproject/galaxy-hub/blob/7fff11d849102f18e2ab5b0a4e07d5881957ff8a/package.json#L22

File: src/assets/overrides.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+// pre styling
+pre.code {
+    white-space: pre-wrap;
+    background: #1d1f21;
+    color: white;
+    padding: 1em;
+}
+
+// Every single line in this file was gotten from the [Galaxy Hub](https://github.com/galaxyproject/galaxy-hub/blob/master/src/assets/styles.scss).
```
Comment (bebatut) [inline src/assets/overrides.scss:88]: Can you move this line for all files on the top?

Comment (rahmot) [reply src/assets/overrides.scss]: Done, thanks! :)

**Comment by bebatut** [comment]
Looks good. Thanks

**Pull request merged**